readme.md

```diff
@@ -29,7 +29,7 @@ This project is for the collaborative effort to build a compelling event managem
 
 We're creating the very network features we need to host events and gather well.
 
-https://www.youtube.com/watch?v=BnYS36C5d38&t=2s
+https://gatherpress.org/demovideo
```

Comment:
This will not render as a video on w.org 

We might have to change this regularly ... I think.

Reply:
Are you sure? For example when pasting such a link on social media, the Youtube embed is rendered

The problem is that I need to wait for the release to make the video, if I want to avoid the version to show -alpha (or it's okay if it shows?), but then we need the new video link every time in the readme at the time of submitting the new version to .org
(or there is a way to have a youtube link in a draft before to upload a video? I'll check that)